### PR TITLE
Add missing privacy keys

### PR DESCRIPTION
**Pull request by gdmarquez90**
These samples crashed in ios10 when the application starts or when trying to use any functionality that requires add privacy key in the .plist file. The keys were added and samples tested.

ios-samples/GLCameraRipple
ios-samples/ios8/ImageInverter
ios-samples/ios8/IntroToHealthKit/HKWork
ios-samples/ios8/PhotoHandoff
ios-samples/ios8/ToastModern
ios-samples/ios8/VideoTimeLine
ios-samples/MediaNotes
ios-samples/PrivacyPrompts
ios-samples/RosyWriter
ios-samples/SharedResources
ios-samples/StateRestoration

### CODE REVIEW

File: PrivacyPrompts/PrivacyPrompts/PrivacyPrompts.sln [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (olegoid) [inline PrivacyPrompts/PrivacyPrompts/PrivacyPrompts.sln]: Quick question here. There is a `PrivacyPrompts_Classic.sln` file in `PrivacyPrompts/PrivacyPrompts` directory. What's the point in having two solution files?

**Comment by olegoid** [comment]
@gdmarquez90 generally looks good. However I have a little request. Could you add a text description for accessibility keys. Something like `XYZ App needs ABC Access` should be fine.

File: PrivacyPrompts/PrivacyPrompts/PrivacyPrompts.sln (hunk 1 above)
Comment (gdmarquez90) [inline PrivacyPrompts/PrivacyPrompts/PrivacyPrompts.sln]: I don't know why there are two solution files. Maybe the IDE creates a new one and accidentally i push it. I'll fix it. Thanks for the feedback.

**Comment by gdmarquez90** [comment]
@olegoid of course i can. i'm on it. Thanks for the feedback.

**Comment by gdmarquez90** [comment]
@olegoid i have added the descriptions.

**Comment by olegoid** [comment]
@gdmarquez90 perfecto! merging to master.

**Pull request merged**